static/app/components/avatar/index.tsx

```diff
@@ -2,18 +2,38 @@ import * as React from 'react';
 
 import OrganizationAvatar from 'app/components/avatar/organizationAvatar';
 import ProjectAvatar from 'app/components/avatar/projectAvatar';
+import SentryAppAvatar from 'app/components/avatar/sentryAppAvatar';
 import TeamAvatar from 'app/components/avatar/teamAvatar';
 import UserAvatar from 'app/components/avatar/userAvatar';
-import {AvatarProject, OrganizationSummary, Team} from 'app/types';
+import {AvatarProject, OrganizationSummary, SentryApp, Team} from 'app/types';
 
 type Props = {
   team?: Team;
   organization?: OrganizationSummary;
   project?: AvatarProject;
+  sentryApp?: SentryApp;
+  /**
+   * True if the Avatar is full color, rather than B&W (Used for SentryAppAvatar)
+   */
+  isColor?: boolean;
+  /**
+   * True if the rendered Avatar should be a static asset
```

Comment:
What does a 'static asset' mean?

Reply:
Not from our DB or filestore, it's a static icon we use in the app, in this case `<IconGeneric>` for our use case.